src/main/java/spoon/support/compiler/jdt/JDTBasedSpoonCompiler.java

```diff
@@ -366,8 +376,13 @@ protected boolean buildTemplates(JDTBuilder jdtBuilder) {
 	 * @param debugMessagePrefix Useful to help debugging
 	 * @return true if the model has been built without errors
 	 */
-	protected boolean buildUnitsAndModel(JDTBuilder jdtBuilder, SpoonFolder sourcesFolder, String[] classpath, String debugMessagePrefix) {
-		CompilationUnitDeclaration[] units = buildUnits(jdtBuilder, sourcesFolder, classpath, debugMessagePrefix);
+	protected boolean buildUnitsAndModel(
+					JDTBuilder jdtBuilder,
+					SpoonFolder sourcesFolder,
+					String[] classpath,
+					List<String> modulePath,
+					String debugMessagePrefix) {
+		CompilationUnitDeclaration[] units = buildUnits(jdtBuilder, sourcesFolder, classpath, modulePath, debugMessagePrefix);
```

Comment:
Isn't this breaking if someone used this class?

Reply:
The question is, is it valuable to add a method without this parameter calling `buildUnitsAndModel` with an empty list. Do you believe anyone directly calls this method?